php-practice/section5.php

```diff
@@ -320,3 +338,19 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $items = [];
+
+  public function addItem($item) {
+    $this->$items[] = $item;
+  }
+
+  public function getItemCount() {
+    $ItemCount = count($items);
+      return $ItemCount;
+  }
```

Comment:
Good job on fixing `addItem`! That's exactly right. Now when we add an item, it goes into the `$this->items` array.

One little gotcha in `getItemCount` -- you did `count($items)`, but we need to do it on `$this->items`. Doing it on just `$items` looks for a local variable (created inside of the function) called `$items`. But that has not been created.

Let me know if that's confusing...